src/main/java/qna/domain/Answer.java

```diff
@@ -1,15 +1,27 @@
 package qna.domain;
 
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
-import java.util.Objects;
+@Entity
+public class Answer extends BaseEntity {
 
-public class Answer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private Long writerId;
     private Long questionId;
+    @Lob
     private String contents;
+
     private boolean deleted = false;
 
```

Comment:
boolean type은 default 값이 있어 `null`이 들어가진 않겠지만 
DDL 생성시 not null 조건을 넣어주면 어떨까요? :)

```suggestion
    @NotNull
    @Column(nullable = false)
    private boolean deleted = false;

```

Reply:
오, 조금더 명시적인 코드를 지향하시는군요 ㅎㅎ 코드 수정하겠습니다.